tests/rules/no-multiple-actions-in-effects.test.ts

```diff
@@ -8,200 +8,171 @@ import { ruleTester } from '../utils'
-      },
-      filename: 'test.ts',
-    },
+        )
+      }`,
+    `
+      @Injectable()
+      export class Effects {
+        effectOK7$ = createEffect(() =>
+          this.actions$.pipe(switchMap(() => {
+            const actions: Action[] = [];
+            return actions.length > 0 ? actions : of(foo());
+          }))
+        )
+      }`,
   ],
   invalid: [
     fromFixture(
       stripIndent`
+        @Injectable()
         export class Effects {
-          one$ = createEffect(() =>
+          effectNOK$ = createEffect(() =>
             this.actions$.pipe(flatMap(_ => [foo(), bar()])),
                                             ~~~~~~~~~~~~~~  [${messageId}]
           )
       }`,
-      {
-        parserOptions: {
-          project: './tsconfig.tests.eslint.json',
-        },
-        filename: 'test.ts',
-      },
     ),
     fromFixture(
       stripIndent`
+        @Injectable()
         export class Effects {
-          two$ = createEffect(() =>
+          effectNOK1$ = createEffect(() =>
             this.actions$.pipe(mergeMap(_ => { return [foo(), bar()] }))
                                                       ~~~~~~~~~~~~~~  [${messageId}]
           )
         }`,
-      {
-        parserOptions: {
-          project: './tsconfig.tests.eslint.json',
-        },
-        filename: 'test.ts',
-      },
     ),
     fromFixture(
       stripIndent`
+        @Injectable()
         export class Effects {
-          three$ = createEffect(() =>
+          effectNOK2$ = createEffect(() =>
             this.actions$.pipe(exhaustMap(function() { return [foo(), bar()] }))
                                                               ~~~~~~~~~~~~~~  [${messageId}]
           )
         }`,
-      {
-        parserOptions: {
-          project: './tsconfig.tests.eslint.json',
-        },
-        filename: 'test.ts',
-      },
     ),
     fromFixture(
       stripIndent`
+        @Injectable()
         export class Effects {
-          readonly ten$ = createEffect(() =>
-            this.actions$.pipe(concatMapTo([test(), baz()]))
-                                           ~~~~~~~~~~~~~~~  [${messageId}]
+          readonly effectNOK3$ = createEffect(() =>
+            this.actions$.pipe(concatMapTo([foo(), bar()]))
+                                           ~~~~~~~~~~~~~~  [${messageId}]
           )
         }`,
-      {
-        parserOptions: {
-          project: './tsconfig.tests.eslint.json',
-        },
-        filename: 'test.ts',
-      },
     ),
     fromFixture(
       stripIndent`
+        const actions = [foo(), bar()]
+        @Injectable()
         export class Effects {
-          eight$ = createEffect(() =>
+          readonly effectNOK4$ = createEffect(() =>
+            this.actions$.pipe(concatMapTo(actions))
+                                           ~~~~~~~  [${messageId}]
+          )
+        }`,
+    ),
+    fromFixture(
+      stripIndent`
+        const actions = [foo(), bar()]
```

Comment:
Just to ensure we report for functions that return `Arrays` as well:

```suggestion
        const actions = () => [foo(), bar()]
```